src/components/title/show/title-show.js

```diff
@@ -250,86 +255,98 @@ class TitleShow extends Component {
             />)
           }
           bodyContent={(
-
-            <Accordion
-              label={<Headline size="large" tag="h3"><FormattedMessage id="ui-eholdings.title.titleInformation" /></Headline>}
-              open={sections.titleShowTitleInformation}
-              id="titleShowTitleInformation"
-              onToggle={this.handleSectionToggle}
-            >
-              <KeyValueColumns>
-                <div>
-                  <ContributorsList data={model.contributors} />
-
-                  {model.edition && (
-                    <KeyValue label={<FormattedMessage id="ui-eholdings.title.edition" />}>
-                      <div data-test-eholdings-title-show-edition>
-                        {model.edition}
-                      </div>
-                    </KeyValue>
-                  )}
-
-                  {model.publisherName && (
-                    <KeyValue label={<FormattedMessage id="ui-eholdings.title.publisherName" />}>
-                      <div data-test-eholdings-title-show-publisher-name>
-                        {model.publisherName}
+            <>
+              <Accordion
+                label={<Headline size="large" tag="h3"><FormattedMessage id="ui-eholdings.title.titleInformation" /></Headline>}
+                open={sections.titleShowTitleInformation}
+                id="titleShowTitleInformation"
+                onToggle={this.handleSectionToggle}
+              >
+                <KeyValueColumns>
+                  <div>
+                    <ContributorsList data={model.contributors} />
+
+                    {model.edition && (
+                      <KeyValue label={<FormattedMessage id="ui-eholdings.title.edition" />}>
+                        <div data-test-eholdings-title-show-edition>
```

Comment:
It is a lit bit hardcoded. `<div>` with data-attribute can be removed. This data-attribute is used for a interactor, but it could be written more simply. `<KeyValue>` component has an interacotr itself in which we can catch the entered value.
This point should not be changed right now, because bigtest will be removed soon, but we must not forgot about it.